website/views.py

```diff
@@ -79,15 +80,13 @@ def index():
     try:
         #TODO : make this way more robust
         inst = Institution.find_one(Q('domain', 'eq', request.host))
```

Comment:
This led me to find that `domain` is a [list field](https://github.com/CenterForOpenScience/osf.io/blob/54418bc5fbce98fdb892267a27a93a560b06433a/website/project/model.py#L3381). Therefore, it should probably be renamed to `domains`.